dagshub/common/logging.py

```diff
@@ -24,13 +24,8 @@ def format(self, record):
 
 
 # create logger with 'spam_application'
-logger = logging.getLogger("DagsHub")
-logger.setLevel(logging.DEBUG)
-
+logger = logging.getLogger()
 # create console handler with a higher log level
 ch = logging.StreamHandler()
```

Comment:
I like the idea of having a pretty printout.
But at the same time, if the user has a root logger already configured, I think this will end up in all the messages double-printing. Once for the library logger, once for the root logger.

<img width="489" alt="image" src="https://user-images.githubusercontent.com/111061261/195551165-3a581b94-b80e-4977-aa41-80c6488caa1f.png">

Reply:
We discussed this I think